apps/storybook/src/overlays/Drawer.stories.tsx

```diff
@@ -1,63 +1,72 @@
 import type { Meta, StoryObj } from "@storybook/react";
 
 import {
-  Button,
   Drawer,
+  DrawerActions,
   DrawerBody,
+  DrawerClose,
+  DrawerContent,
+  DrawerDescription,
   DrawerFooter,
+  DrawerHeader,
   DrawerTitle,
-  Flex,
+  DrawerTrigger,
 } from "@optiaxiom/react";
-import { type ComponentPropsWithoutRef, useState } from "react";
+import { type ComponentPropsWithoutRef, type ReactNode } from "react";
 
-const withTemplate = ({
-  content = "This is the Drawer content.",
-  defaultOpen = true,
-  description = "",
-  withFooter = false,
-} = {}) =>
-  function Template(props: Partial<ComponentPropsWithoutRef<typeof Drawer>>) {
-    const [open, setOpen] = useState(defaultOpen);
-
-    const onOpen = () => setOpen(true);
-    const onClose = () => setOpen(false);
+type DrawerStoryProps = {
+  actions?: ReactNode;
+  content?: string;
+  description?: string;
+  withFooter?: boolean;
+} & ComponentPropsWithoutRef<typeof Drawer> &
+  Pick<ComponentPropsWithoutRef<typeof DrawerContent>, "position">;
 
+export default {
+  args: {
+    content: "This is the drawer content.",
+    defaultOpen: true,
+  },
+  component: Drawer,
+  render: function Template({
+    actions,
+    content,
+    description,
+    withFooter,
+    ...args
+  }) {
     return (
-      <Flex>
-        <Button onClick={onOpen}>Open Drawer</Button>
+      <Drawer {...args}>
+        <DrawerTrigger>Open Drawer</DrawerTrigger>
+
+        <DrawerContent {...args}>
```

Comment:
Vai, when we don't pass any description, it will show a warning, we handled this in dialog like this way.

```suggestion
        <DrawerContent 
        {...(!description && { ["aria-describedby"]: undefined })}
         {...args}
         >
```